avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroSerDeConfig.java

```diff
@@ -61,4 +69,8 @@ public int getMaxSchemasPerSubject() {
     return this.getList(SCHEMA_REGISTRY_URL_CONFIG);
   }
 
+  public boolean autoRegister() {
```

Comment:
`getAutoRegister` is better i think

Reply:
.. but get is used with other properties in this class?

Reply:
the general approach is to avoid `get` and `set` prefixes, but they occasionally leak in, especially in early code where not everyone was familiar w/ the Kafka conventions